src/poetry/mixology/incompatibility.py

```diff
@@ -256,7 +242,7 @@ def and_to_string(
         if this_line is not None:
             buffer.append(" " + str(this_line))
```

Comment:
I think we might as well turn these in to f-strings (with `!s`) for consistency.